csi: move csi management to admin (#17176)
rook/rook: Documentation/Helm-Charts/helm-charts.md

title: Helm Charts Overview
---

> Rook has published the following Helm charts for the Ceph storage provider. Install and configure them in this order when you use Helm end to end:

Reviewer: Rook doesn't publish the drivers chart, so let's just simplify this statement
```suggestion
The following charts are available to configure Ceph storage:
```

> 1. [Rook Ceph Operator](operator-chart.md): Starts the Ceph Operator, which will watch for Ceph CRs (custom resources), and installs the Ceph-CSI operator subchart by default (`csi.rookUseCsiOperator`) so its CRDs are available.

Reviewer: Seems like we should always install the CSI operator, and no need for the rookUseCsiOperator setting.
```suggestion
1. [Rook Ceph Operator](operator-chart.md): Starts the Ceph Operator, which will watch for Ceph CRs (custom resources). Also installs the Ceph-CSI operator as a helm dependency.
```

> 1. [Ceph-CSI drivers chart](csi-drivers-chart.md): Installs the [Ceph-CSI Driver chart](https://github.com/ceph/ceph-csi-operator/blob/main/docs/helm-charts/drivers-chart.md) from the Ceph-CSI Operator project so the CSI drivers are configured to provision and mount volumes.

Reviewer: ```suggestion
1. [Ceph-CSI drivers chart](csi-drivers-chart.md): Installs the Ceph-CSI drivers to provision and mount volumes.
```
